app/views/overview/_service-instance-row.html

```diff
@@ -101,8 +101,7 @@
               <div class="alert word-break alert-warning">
                 <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
                 <span class="sr-only">warning</span>
-                <span class="strong">The service was marked for deletion</span>
-                <span class="strong" am-time-ago="row.apiObject.metadata.deletionTimestamp"></span>.
+                <span class="strong">The service was marked for deletion</span>.
```

Comment:
nit: Any reason not to put the period inside the span so it has the strong style like the rest of the sentence?

Reply:
no reason at all... I've realized it after hitting the Open PR btn... fixing atm